boards.txt

```diff
@@ -555,6 +556,9 @@ GenF103.menu.upload_method.bmpMethod=BMP (Black Magic Probe)
 GenF103.menu.upload_method.bmpMethod.upload.protocol=gdb_bmp
 GenF103.menu.upload_method.bmpMethod.upload.tool=bmp_upload
 
+GenF103.menu.upload_method.bmpMethod=HID Bootloader 2.0 (0x0800)
```

Comment:
```suggestion
GenF103.menu.upload_method.HIDBootloaderMethod=HID Bootloader 2.0 (0x0800)
```

Reply:
I'll fix it shortly - it still works however... as if by magic.